package.json

```diff
@@ -49,11 +55,12 @@
     "template"
   ],
   "scripts": {
-    "test": "NODE_ENV=test mocha --reporter spec --timeout 30s --bail test/app/index.js",
-    "test-cov": "NODE_ENV=test node node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha -- --reporter dot --timeout 2m --bail test/app/index.js",
-    "test-travis": "NODE_ENV=test node node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha --report lcovonly -- --reporter dot --timeout 2m --bail test/app/index.js",
+    "test": "NODE_ENV=test mocha --reporter spec --timeout 30s --bail test/**/*.js",
+    "test-cov": "NODE_ENV=test istanbul cover mocha -- --reporter dot --timeout 2m --bail test/**/*.js",
+    "test-travis": "NODE_ENV=test istanbul cover mocha --report lcovonly -- --reporter dot --timeout 2m --bail test/**/*.js",
```

Comment:
Unfortunately using `mocha` breaks the coverage reporting because the default `mocha` executable runs in another process (out of band), so istanbul can't tell what's going on. That's why we had `_mocha` here, which runs in the same process as a workaround. Note that if we switch to nyc (istanbul's new CLI) we may not need to do this.

I agree with removing `./node_modules/.bin` and using `test/**/*.js` but let's remove the automatic requiring in `test/app/index.js` too.

Reply:
@nickmccurdy Good catch here. `jest` has built-in coverage reports. We will be replacing all of those scripts soon, anyway.

Reply:
Fixed